libs/pages/application/src/lib/ui/page-deployments/page-deployments.spec.tsx

```diff
@@ -1,5 +1,5 @@
 import { render } from '__tests__/utils/setup-jest'
-import { DeploymentsPage, DeploymentsProps } from './deployments-page'
+import { DeploymentsProps, PageDeployments } from './page-deployments'
```

Comment:
`DeploymentsProps` should be `PageDeploymentsProps` ?

Reply:
good catch!